README.md

```diff
@@ -433,12 +433,13 @@ These may be specified with the `--format` flag. For example, `--format=json`.
 
 The following checks are all run against the target project by default:
 
+[CII-Best-Practices](docs/checks.md#cii-best-practices)         | Has the project earned an [OpenSSF (formerly CII) Best Practices Badge](https://bestpractices.coreinfrastructure.org) at the passing, silver, or gold level?                                                                                                                                                                                                                         | Low  | PAT, GITHUB_TOKEN   |
```

Comment:
This looks like a copy paste error?

Reply:
Hmm, only 1 intended. Let me try to tweak that...!!

Reply:
I think I fixed it now, thanks!